bin/deploy-vhost

```diff
@@ -1056,6 +1057,45 @@ sub remove_site {
 
 }
 
+# Upgrade database password on the fly
+sub upgrade_db_password {
+    my ($params, $database) = @_;
+
+    my $db_host = get_db_host ($params, 0);
```

Comment:
This might just be for my benefit, but I'd prefer no space between function name and opening bracket, as then it's easier to search for calls to the function :)

Reply:
Makes sense, will fix.